client/src/main/java/com/netflix/conductor/client/http/TaskClient.java

```diff
@@ -218,9 +218,13 @@ public Task getPendingTaskForWorkflow(String workflowId, String taskReferenceNam
      * it is uploaded to {@link ExternalPayloadStorage}, if enabled, else the task is marked as FAILED_WITH_TERMINAL_ERROR.
      *
      * @param taskResult the {@link TaskResult} of the executed task to be updated.
-     * @param taskType   the type of the task
      */
-    public void updateTask(TaskResult taskResult, String taskType) {
+    public void updateTask(TaskResult taskResult) {
```

Comment:
The contract of this public method has changed from: 
`public void updateTask(TaskResult taskResult, String taskType)` to `public void updateTask(TaskResult taskResult)`, does this break backward compatibility for the existing usage ? 

If that is the case, I would suggest leave the existing method and mark it as deprecated and create a new overloaded method